src/main/java/uk/gov/gchq/palisade/service/request/ResourceDetails.java

```diff
@@ -63,44 +79,34 @@ private static Matcher validateNameRegex(final String fileName) {
         return FILENAME_PATTERN.matcher(fileName);
     }
 
-    public String getFileName() {
-        return fileName;
-    }
-
-    public String getType() {
-        return type;
-    }
-
-    public String getFormat() {
-        return format;
-    }
-
     @Override
+    @Generated
```

Comment:
update equals method to use new equals method which uses instanceOf

Reply:
Ok, done